### PR TITLE
Story1

### PR DESCRIPTION
**Pull request by HarryCusworth**
files in place, sql database and images in final format.

### CODE REVIEW

File: src/Biscuit.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace BiscuitApp;
+class Biscuit
+{
+
+}
```
Comment (norayna) [inline src/Biscuit.php]: missing:
```php
require_once 'vendor/autoload.php';
```

also in the other two classes

Comment (norayna) [reply src/Biscuit.php]: might be slightly different path

Comment (HarryCusworth) [reply src/Biscuit.php]: resolved

**Comment by norayna** [comment]
put sql file in a folder called `db` in the main project directory

File: src/composer.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "autoload": {
+        "psr-4": {
+            "BiscuitApp\\": "."
+        }
+    },
```
Comment (norayna) [inline src/composer.json]: put this bit on the end (after `require-dev`) just for tidiness so the file starts with `name` `description` etc

File: src/test/BiscuitDisplayerTest.php [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace BiscuitApp\Test;
+class BiscuitDisplayerTest
+{
+
+}
```
Comment (norayna) [inline src/test/BiscuitDisplayerTest.php]: for these test files you need to extend TestCase (PHPUnit class) and you need to import that class. do that in all 3 files

**Review by norayna — CHANGES_REQUESTED** [review]
please address the previous comments and let me know when I should review again

**Comment by norayna** [comment]
oh also need require vendor autoload in test files as well

File: src/composer.json (hunk 2 above)
Comment (HarryCusworth) [reply src/composer.json]: resolved

**Comment by Emily774** [comment]
> put sql file in a folder called `db` in the main project directory

resolved

File: src/test/BiscuitDisplayerTest.php (hunk 3 above)
Comment (HarryCusworth) [reply src/test/BiscuitDisplayerTest.php]: resolved

File: src/test/BiscuitDisplayerTest.php [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace BiscuitApp\Test;
+use PHPUnit\Framework\TestCase;
+require_once '../vendor/autoload.php';
+
+class BiscuitDisplayerTest extends TestCase
+{
+
+}
```
Comment (norayna) [inline src/test/BiscuitDisplayerTest.php]: put `namespace` first, then empty line, then `require_once`, then empty line, then `use`

Comment (norayna) [reply src/test/BiscuitDisplayerTest.php]: in all three files

File: src/index.php [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+namespace BiscuitApp;
```
Comment (norayna) [inline src/index.php]: delete this line - no need to declare namespace in file that isn't class/interface/trait

File: db/biscuits.sql [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+DROP TABLE IF EXISTS `biscuits`;
+
+CREATE TABLE `biscuits` (
+  `name` varchar(19) NOT NULL,
+  `img` varchar(181) NOT NULL,
+  `RDT` int(11) NOT NULL,
+  `description` varchar(355) NOT NULL,
+  `wikipedia` varchar(68) NOT NULL,
+  `ID` int(11) NOT NULL AUTO_INCREMENT,
```
Comment (norayna) [inline db/biscuits.sql]: ID is on the end again - will appear as last column, this was already fixed in a previous commit

File: src/test/BiscuitDisplayerTest.php (hunk 4 above)
Comment (HarryCusworth) [reply src/test/BiscuitDisplayerTest.php]: resolved

File: db/biscuits.sql (hunk 6 above)
Comment (HarryCusworth) [reply db/biscuits.sql]: resolved

File: src/index.php (hunk 5 above)
Comment (HarryCusworth) [reply src/index.php]: resolved

**Review by norayna — APPROVED** [review]

**Pull request merged**